tyrian/js/src/main/scala/tyrian/Cmd.scala

```diff
@@ -1,67 +1,63 @@
 package tyrian
 
+import cats.effect.kernel.Sync
+
+import scala.annotation.targetName
+
 /** A command describes some side-effect to perform.
-  *
-  * The difference with a `Task` is that a command never produces error values.
   */
-sealed trait Cmd[+Msg]:
-  def map[OtherMsg](f: Msg => OtherMsg): Cmd[OtherMsg]
+sealed trait Cmd[+F[_], +Msg]:
+  /** Transforms the type of messages produced by the command */
+  def map[OtherMsg](f: Msg => OtherMsg): Cmd[F, OtherMsg]
+
+object Cmd:
+  given CanEqual[Cmd[_, _], Cmd[_, _]] = CanEqual.derived
+
+  extension [F[_], Msg, LubMsg >: Msg](cmd: Cmd[F, Msg])
+    def combine(other: Cmd[F, LubMsg]): Cmd[F, LubMsg] = Cmd.merge(cmd, other)
+    def |+|(other: Cmd[F, LubMsg]): Cmd[F, LubMsg]     = Cmd.merge(cmd, other)
```

Comment:
I think you can put these directly on `Cmd` by moving the `LubMsg` parameter to the method. I think this helps API discoverability for users and is easier on the compiler as well.

Also I guess this is a Monoid? That's pretty cool :)

Reply:
I was struggling with that, it used to live on `Cmd`. It wasn't the `LubMsg` that was causing me problems, it was the covariant `F`. Help most welcome.

Yes, `Sub` and `Cmd` are monoids. I love monoids. You should see Indigo - it's monoid-tastic! 😁 

Originally there were cats monoid instances of these available for use, but I removed them when I removed cats. Could bring them back now if there was any value in it.

Reply:
Ah, right, you need to do the exact same thing for `F[_]` too :) e.g. see the signatures in fs2:
```scala
  def concurrently[F2[x] >: F[x], O2](
      that: Stream[F2, O2]
  )(implicit F: Concurrent[F2]): Stream[F2, O] = {
```

Reply:
Works like a charm! Have pushed the change.

I did try that -except- that I didn't know about the little trick with the `x` you've got there. If you just use `_` you get an error: `unreducible application of higher-kinded type F to wildcard arguments`

Thank you kindly. 😄

Reply:
Monoid instances are also back. I even added tests.